tools/mod/go.mod

```diff
@@ -1,8 +1,8 @@
 module go.etcd.io/etcd/tools/v3
 
-go 1.22
+go 1.22.1
 
-toolchain go1.22.6
+toolchain go1.23.0
```

Comment:
Hi @henrybear327, I think the verify job is failing due to this change.

Reply:
Confirmed to be the issue! The new commit is pushed :) Thanks @ivanvc